config/config.go

```diff
@@ -342,6 +345,9 @@ type Cluster struct {
 
 	// Catches all undefined fields
 	XXX map[string]interface{} `yaml:",inline"`
+
+	// Retry number for query - how many times a query can retry after bad gateway response from the host
+	RetryNumber int `yaml:"retrynumber"`
```

Comment:
to stay consistent with other fields
```suggestion
	RetryNumber int `yaml:"retry_number"`
```

Reply:
Done